src/titlepane/TitlePane.ts

```diff
@@ -73,6 +79,30 @@ export class TitlePaneBase<P extends TitlePaneProperties = TitlePaneProperties>
 		}
 	}
 
+	protected animate(effects: any[]) {
+		const { open = true } = this.properties;
+		this.meta(WebAnimation).animate('content', {
```

Comment:
i'd imagine you'll need to add an `onAnimationEnd` callback to set `play` to false btw. Could be wrong but i had issues with this elsewhere when other things in the page re-render.